src/examples/rotation.app.json

```diff
@@ -0,0 +1,579 @@
+                                    "type": "Mesh",
+                                    "name": "Si-0",
+                                    "layers": 1,
+                                    "matrix": [
+                                        0.30000000000000004,
+                                        0,
+                                        0,
+                                        0,
+                                        0,
+                                        0.30000000000000004,
+                                        0,
+                                        0,
+                                        0,
+                                        0,
+                                        0.30000000000000004,
+                                        0,
+                                        0,
+                                        0,
+                                        0,
+                                        1
+                                    ],
+                                    "geometry": "a42ab9b0-1400-4f51-8f60-39c948613f3c",
+                                    "material": "b1557272-29d9-4bc1-a2ba-14c28b9bf4c6"
+                                },
+                                {
+                                    "uuid": "1a84c09f-42a4-4e67-9801-0f7419ec021f",
+                                    "type": "Mesh",
+                                    "name": "Si-1",
+                                    "layers": 1,
+                                    "matrix": [
+                                        0.30000000000000004,
+                                        0,
+                                        0,
+                                        0,
+                                        0,
+                                        0.30000000000000004,
+                                        0,
+                                        0,
+                                        0,
+                                        0,
+                                        0.30000000000000004,
+                                        0,
+                                        1.1163067454781415,
+                                        0.7893480696118791,
+                                        1.9335000000000002,
+                                        1
+                                    ],
+                                    "geometry": "a42ab9b0-1400-4f51-8f60-39c948613f3c",
+                                    "material": "a442a002-f739-4802-a2ee-0f71baf176ae"
+                                }
+                            ]
+                        }
+                    ]
+                }
+            ]
+        }
+    },
+    "scripts": {
+		"fd0d59b8-d1a5-44bd-b8b2-591949fbaa00": [
+			{
+				"name": "Rotate",
+				"source": "/* eslint-disable-next-line */\nfunction update(event) { \n\tthis.rotation.x += 0.1;\n }"
```

Comment:
Do we really need the eslint comment here??

Reply:
Yes, because I enabled eslint in codemirror and this function is not used anywhere.